plf-packaging-resources/src/main/resources/CHANGE_LOG.txt

```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [ECMS-7940] - Broken Document thumbnail
-    * [ECMS-7934] - Compare button is not functional
+    * [ECMS-7818] - Exception when search for a word existing in an uploaded file 
 	
 eXo Social ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [SOC-6244] - No sort applied in "My connexions", "Invitation" & "Pending" tabs of People directory
-    * [SOC-6237] - Disable useless MaxQueries check in UT
-    * [SOC-6145] - Sorting in People Directory doesn't work when filtering on a letter
+    * [SOC-6175] - Loading Icon is still displayed when click on Load More button
 
-** Improvement
-    * [SOC-6239] - Make the identities sort field extensible
     
 eXo Calendar ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [CAL-1361] - Nothing appears on Calendar when creating a task with views Board
+    * [CAL-1464] - Space's Calendar preferences
```

Comment:
I changed description to `Ignore selected calendars user preference in space calendar app`